invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar/communities.html

```diff
@@ -0,0 +1,27 @@
+<section id="sidebar-communtities" aria-label="{{ _('Record communities') }}" class="ui segment rdm-sidebar">
+  <div class="header flex align-items-center justify-space-between rel-mb-1">
+    <h2 class="ui small header m-0">{{ _('Communities')}}</h2>
+
+    <!-- TODO: To be added in a different PR where management modals are implemented -->
+    <!-- {% if can_manage_record %}
+      <div class="ui dropdown">
+        <button aria-label="{{ _('Open community management menu') }}" aria-expanded="false" class="ui transparent icon button flex align-items-center">
+          <i class="ui cog icon" aria-hidden="true"></i>
+          <i class="ui dropdown icon" aria-hidden="true"></i>
+        </button>
+        <ul class="menu">
+          <li role="button" id="submit-to-community-modal" class="item flex align-items-center">
+            {# Trigger defined by React #}
+          </li>
+        </ul>
+      </div>
+    {% endif %} -->
+  </div>
+
+  {# Communities list #}
+  {#  TODO: Update endpoint to {{ record.links["communities"] }} specific #}
+  {%  set ids = "id:" + record.parent.communities.ids | join (' OR id:')  %}
+  <div id="record-communities-list" data-record-community-search-endpoint="/api/communities?q={{ ids }}"></div>
```

Comment:
I think that we should not use the `q`, because if in the future we add a search bar, then the query will be displayed and the user might be able to change them.
There was this `hidden` field at some point or we can find better solution.

Reply:
this will be replaced with result.links["communities"], it is temporary